packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -163,6 +165,18 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
 
         val dataStoreConfigurationBuilder = DataStoreConfiguration.builder()
 
+        if( (request["hasErrorHandler"] as? Boolean? == true) ) {
```

Comment:
I like the idea that defining a default error handler like what you did in this PR for the iOS implementation, can we adapt it to the Android implementation as well?

Reply:
Good point !  Updated android: 

  ```
      var errorHandler : DataStoreErrorHandler;
        errorHandler = if( (request["hasErrorHandler"] as? Boolean? == true) ) {
            DataStoreErrorHandler {
                val args = hashMapOf(
                        "errorCode" to "DataStoreException",
                        "errorMessage" to ExceptionMessages.defaultFallbackExceptionMessage,
                        "details" to createSerializedError(it)
                )
                channel.invokeMethod("errorHandler", args)
            }
        }
        else {
            DataStoreErrorHandler {
                LOG.error(it.toString())
            }
        }
        dataStoreConfigurationBuilder.errorHandler(errorHandler)

```